### PR TITLE
chore(security): Support Go Vulnerability Management

### PR DESCRIPTION
**Pull request by appleboy**
See https://go.dev/security/vuln/

Signed-off-by: Bo-Yi.Wu <appleboy.tw@gmail.com>

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -39,6 +39,16 @@ steps:
       - make lint-frontend
     depends_on: [deps-frontend]
 
+  - name: security-check
+    image: golang:1.18
```
Comment (techknowlogick) [inline .drone.yml]: Can this be golang 1.19?

Comment (appleboy) [reply .drone.yml]: Done dbaa18d5

**Review by techknowlogick — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Comment by silverwind** [comment]
We can probably move this into golangci-lint after https://github.com/golangci/golangci-lint/pull/3199.

**Pull request merged**